src/functionaltests/resources/AggregatedObject.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
I think you have these files under "src/test/resources/..." . I think its better to reuse files instead of duplicate files. Or have you done and any specific changes in the files for your tests?
This comment applies to the other json files also.